glyphs-reader/src/font.rs

```diff
@@ -145,24 +150,48 @@ impl Glyph {
 
 impl FromPlist for Node {
     fn from_plist(plist: Plist) -> Self {
-        let mut spl = plist.as_str().unwrap().splitn(3, ' ');
-        let x = spl.next().unwrap().parse().unwrap();
-        let y = spl.next().unwrap().parse().unwrap();
-        let pt = Point::new(x, y);
-        let node_type = spl.next().unwrap().parse().unwrap();
-        Node { pt, node_type }
+        match &plist {
+            Plist::String(value) => {
+                let mut spl = value.splitn(3, ' ');
+                let x = spl.next().unwrap().parse().unwrap();
+                let y = spl.next().unwrap().parse().unwrap();
+                let pt = Point::new(x, y);
+                let node_type = spl.next().unwrap().parse().unwrap();
+                Node { pt, node_type }
+            }
+            Plist::Array(value) => {
```

Comment:
should we not check the .glyphs file's `formatVersion` to conditionally parse nodes as strings or arrays?
Here you're parsing either string or array regardless of format version, which will work but I'd argue would not be "legal"

https://github.com/schriftgestalt/GlyphsSDK/blob/Glyphs3/GlyphsFileFormat/GlyphsFileFormatv3.md#element-tree
> .formatVersion int set to 3 for version 3. If that key is missing assume version 2.

Reply:
I'd much prefer to keep the logic here "dumb" if possible. I don't think the representations are ambiguous so it's OK.